### DISCUSSION TITLE
QS-Zigbee-c01

### DISCUSSION THREAD

**Discussion by wanaka1975** [Koenkk/zigbee2mqtt#8688] [Devices]
I noticed my curtain module QS-Zigbee-c01 is detected as TS130F.  I am running home assistant with zigbee2mqtt 1.21.1-1.
How can I fix this?  I have tried unpair and pair again but result is the same.

**Comment by okastl** [comment]
Zigbee model TS130F is correct. Important is the zigbee manufacturer, in my case _TZ3000_vd43bbfq.
If yours is different, it must be added to lonsonho.js.

**Comment by wanaka1975** [comment]
mine is shown as _TZ3000_fccpjz5z.   Is this considered wrong?  I have issued with calibration

**Reply by Koenkk** [reply]
Added this, 
Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html)
